Tools/tsan/suppressions_free_threading.txt

```diff
@@ -43,6 +43,10 @@ race_top:write_thread_id
 race_top:PyThreadState_Clear
 # Only seen on macOS, sample: https://gist.github.com/aisk/dda53f5d494a4556c35dde1fce03259c
 race_top:set_default_allocator_unlocked
+# See https://github.com/python/cpython/pull/128147
```

Comment:
I think the comment should refer to https://github.com/python/cpython/issues/128421

Reply:
Sorry that was typo, fixed now